prody/proteins/interactions.py

```diff
@@ -2410,7 +2416,7 @@ def showFrequentInteractors(self, cutoff=5, **kwargs):
             fig = plt.figure(num=None, figsize=(12,6), facecolor='w')
         
         y_pos = np.arange(len(y))
-        show = plt.bar(y_pos, x, align='center', alpha=0.5, color='blue', **kwargs)
```

Comment:
why? Users may want to set bar kwargs such as width or shading

Reply:
I know, but kwargs are also needed to add to the dictionary non-standard residues to include them.        
for key, value in kwargs.items():
            aa_dict[key] = value
(line 2393) 
I want to add kwargs in plt.bar as well, but they will be in conflict with non-standard names. Any idea how to solve it? I have for example HSD1, HSD2 etc, and my colleague CYX. Other people may also have non-standard residues/names.

Reply:
ok, fair point. Probably, it's too difficult to control

Reply:
Non-standard may have 3 or 4 letters. Bigger will not fit the PDB standards. If kwargs for plt.bar will have more than >4 Ietters then maybe like this?

Reply:
Unfortunately not. They have ones like log and xy. Let's just leave it non-customisable, but maybe put something in the docs.

Reply:
Ok.